kubeflow/tf-serving/tf-serving-service-template.libsonnet

```diff
@@ -0,0 +1,99 @@
+{
+  local k = import "k.libsonnet",
+  local util = import "kubeflow/tf-serving/util.libsonnet",
+  new(_env, _params):: {
+    local params = _env + _params {
+      namespace: if std.objectHas(_params, "namespace") && _params.namespace != "null" then
+        _params.namespace else _env.namespace,
+    },
+    local namespace = params.namespace,
+    local name = params.name,
+    local modelName =
+      if params.modelName == "null" then
+        params.name
+      else
+        params.modelName,
+
+    local tfService = {
+      apiVersion: "v1",
+      kind: "Service",
+      metadata: {
+        labels: {
+          app: modelName,
+        },
+        name: name,
+        namespace: namespace,
+        annotations: {
+          "getambassador.io/config":
+            std.join("\n", [
+              "---",
+              "apiVersion: ambassador/v0",
+              "kind:  Mapping",
+              "name: tfserving-predict-mapping-" + modelName,
+              "prefix: /tfserving/models/" + modelName,
```

Comment:
We don't need to address it in this PR; but do we need to have individual rewrite rules for different paths and methods?

Could we just have a single rewrite rule
/tfserving/models/modelName -> /v1/models/